applications/nfc/nfc_worker.c

```diff
@@ -613,14 +613,16 @@ void nfc_worker_emulate_mifare_ul(NfcWorker* nfc_worker) {
     // Setup emulation parameters from mifare ultralight data structure
     mf_ul_prepare_emulation(&mf_ul_emulate, &data->mf_ul_data);
     while(nfc_worker->state == NfcWorkerStateEmulateMifareUl) {
+        // WARNING
+        // DO NOT call any blocking functions (e.g. FURI_LOG_*) in this loop,
+        // as any delay will negatively affect the stability of the emulation.
         if(furi_hal_nfc_listen(
                data->nfc_data.uid,
                data->nfc_data.uid_len,
                data->nfc_data.atqa,
                data->nfc_data.sak,
                true,
                200)) {
-            FURI_LOG_D(TAG, "Anticollision passed");
```

Comment:
I guess we can with logging level to debug and leave it here. @gornekich what do you think?

Reply:
I agree, it's better to leave logs at Debug level. @ghettorce could you make it?